core/modules/mysql/MySqlConfig.h

```diff
@@ -46,34 +46,37 @@ class MySqlConfig {
     /**
      *  Create MySqlConfig instance
      *
-     *  @param username:    MySQL username
-     *  @param password:    MySQL password
-     *  @param hostname:    MySQL hostname
-     *  @param port:        MySQL port
-     *  @param socket:      MySQL socket
-     *  @param db:          MySQL database
-     *  @param checkValid:  if true, throws exception if parameters below are invalid
+     *  @param username:     MySQL username
+     *  @param password:     MySQL password
+     *  @param hostname:     MySQL hostname
+     *  @param port:         MySQL port
+     *  @param socket:       MySQL socket
+     *  @param db:           MySQL database
+     *  @param maxtablesize: maximum table size.
      *
      * @throws std::runtime_error: if checkValid is true and some parameters are invalid
      */
     MySqlConfig(std::string const& username, std::string const& password,
                 std::string const& hostname,
                 unsigned int const port,
                 std::string const& socket,
-                std::string const& db = "");
+                std::string const& db = "",
+                size_t maxresultsize = 0);
```

Comment:
I'm not terribly thrilled with this. maxresultsize only means something for resultDb, which is why it defaults to zero here. It might better to create a child configuration class just for the result database.